modules/integration/tests-integration/tests-backend/src/test/resources/script.js

```diff
@@ -0,0 +1,21 @@
+function test(context) {
+    var acr = selectAcrFrom(context, ["acr1", "acr2"]);
+    Log.info("--------------- ACR selected "+ acr);
+    //comment
+    switch(acr) {
+        case "acr1" : executeStep({id :'1',
+            on : {
+                success : function(context) {
+                    var isAdmin = hasRole(context, 'admin');
+                    Log.info("--------------- Has Admin "+isAdmin);
+                    if(isAdmin) {
+                        executeStep({id :'2'});
+                    }
+                }
+            }
+        });
+            break;
+        case "acr2" : executeStep({id :'1'}); executeStep({id :'2'});  break;
+        default :  executeStep({id :'1'});  executeStep({id :'2'});
+    }
+}
```

Comment:
EOF

Reply:
We need to put this file inside src/test/resources/org/wso2/identity/integration/test/auth and rename to the test class. ConditionalAuthenticationTestCase.js